src/styles/stars.scss

```diff
@@ -0,0 +1,34 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  &--0 &__star:nth-child(n),
+  &--1 &__star:nth-child(n + 2),
+  &--2 &__star:nth-child(n + 3),
+  &--3 &__star:nth-child(n + 4),
+  &--4 &__star:nth-child(n + 5) {
+    background-image: $star-bg;
```

Comment:
The variable `$star-bg` is used here for the background image. Ensure that this variable is defined in the appropriate SCSS file to avoid compilation errors.